pemFioi/conceptViewer-1.0-mobileFirst.js

```diff
@@ -130,36 +138,56 @@ var conceptViewer = {
     navLanguage += '</ul>';
 
     $('body').append(''
-      + '<div id="conceptViewer" style="display: none;">'
-      + '  <div class="content">'
-      + '   <div class="panel-heading">'
-      + '     <h2 class="sectionTitle"><span class="icon fas fa-list-ul"></span>' + this.strings.viewerTitle + '</h2>'
-      + '     <div class="exit" onclick="conceptViewer.hide();"><span class="icon fas fa-times"></span></div>'
-      + '   </div>'
-      + '   <div class="panel-body">'
-      + '     <div class="navigation">'
-      + '      <div class="navigationLanguage">'
-      + navLanguage
-      + '      </div>'
-      + '      <div class="navigationContent"></div>'
-      + '    </div>'
-      + '    <div class="viewer">'
-      + '      <iframe class="viewerContent" name="viewerContent"></iframe>'
-      + '    </div>'
-      + '   </div>'
-      + '  </div>'
-      + '</div>');
+        + '<div id="conceptViewer"">'
+        + '  <div class="content">'
+        + '   <div class="panel-heading">'
+        + '     <h2 class="sectionTitle"><span class="icon fas fa-list-ul"></span>'
+        +         (this.fullScreen ? (this.contextTitle ? this.contextTitle + " &ndash; " : "") : "") + this.strings.viewerTitle
+        + '     </h2>'
+        + '     <div class="section-external-exit">'
+        + '       <div class="exit" onclick="conceptViewer.openInNewWidget();"><span class="icon fas fa-external-link-alt"></span></div>'
+        + '       <div class="exit" onclick="conceptViewer.hide();"><span class="icon fas fa-times"></span></div>'
+        + '     </div>'
+        + '   </div>'
+        + '   <div class="panel-body">'
+        + '     <div class="navigation">'
+        + '      <div class="navigationLanguage">'
+        + navLanguage
+        + '      </div>'
+        + '      <div class="navigationContent"></div>'
+        + '    </div>'
+        + '    <div class="viewer">'
+        + '      <iframe class="viewerContent" name="viewerContent"></iframe>'
+        + '    </div>'
+        + '   </div>'
+        + '  </div>'
+        + '</div>');
+    if (!this.fullScreen) {
+      $('#conceptViewer').hide();
+    } else {
+      $('#conceptViewer .content').css({
```

Comment:
You can replace lines 168-175 by putting all that in the associated CSS file, and adding a class to `#conceptViewer`.

You would just have to do, for instance, `$('#conceptViewer').addClass('conceptViewer-fullscreen');` and then in the CSS file you just have a selector `#conceptViewer.conceptViewer-fullscreen .content {}` with all the rules you put there. In the same way you'd get `#conceptViewer.conceptViewer-fullscreen .section-external-exit { display: none; }` instead of using `.hide()`.

Reply:
For the full screen, the css have been written in conceptViewer-1.0-mobileFirst.css